app/assets/stylesheets/application.scss

```diff
@@ -44,16 +44,36 @@ p{
   font-family: 'Caveat Brush', cursive;
 }
 
-
 .loginBtn--twitter {
   position: relative;
   text-align: center;
   }
-
-
 .loginBtn--twitter:before {
   background: url('https://s.gr-assets.com/assets/icons/twitter_icon-2289166aec1c47ee68cfb5716142273f.jpg') 6px 6px no-repeat;
 }
 .loginBtn--twitter:focus {
   background: #66ccff;
 }
+.item {
+  height: 650px;
+  border-style: solid;
+  border-width: 2px;
+  border-radius: 5px;
+  background-color: white;
+}
+.item_text {
+  height: 150px;
+}
+.item_title {
+  height: 60px;
+  font-family: 'Caveat Brush', cursive;
+  background: linear-gradient(rgba(0, 0, 0, 0.3), white);
```

Comment:
@liambarstad I'm going to need you to demo this font just so I can determine that it goes with the overall website.